README.md

```diff
@@ -30,5 +30,274 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r21.09 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.9.0
+
+* Full-featured, Beta version of
+  [Business Logic Scripting (BLS)](https://github.com/triton-inference-server/python_backend/tree/r21.09#business-logic-scripting-beta)
+  released.
+
+* Beta version for basic JAVA Client released. See
+  <https://github.com/triton-inference-server/client/tree/r21.09/src/java> for a
+  list of supported features.
+
+* A stack trace is now printed when Triton crashes to aid in debugging.
+
+* The [Triton Client SDK](https://github.com/triton-inference-server/client/tree/r21.09#download-using-python-package-installer-pip)
```

Comment:
The client SDK docs still document nvidia-pyindex but not pypi.
Additionally https://pypi.org/project/tritonclient links to the main branch instead of a release branch. We should use the respective release branch instead.
Additionally the description could be enhanced on the pypi portal. Right now it just links to the github docs

Reply:
Does this affect this particular statement? It sounds like changes are needed on the client documentation, not here in this readme

Reply:
@tanmayv25  Seems like this PR just needs to be cherry-picked into r21.09 branch: https://github.com/triton-inference-server/client/pull/33

Reply:
@dzier Yes these changes look good but we should improve the client docs to reflect the pypi changes

Reply:
Yes.. cherry-picking [#33](triton-inference-server/client#33)  to 21.09 should fix the nvidia-pyindex issue. It is essentially a documentation fix so should not carry risk or require us to re-spin.

> Additionally the description could be enhanced on the pypi portal. Right now it just links to the github docs.
I believe it is better to handle documentation at one place than handling it on multiple.